src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -93,6 +95,32 @@ object DUOS {
     }
   }
 
+
+  case class StructuredDataRequest(generalResearchUse: Boolean,
+                                   healthMedicalBiomedicalUseOnly: Boolean,
+                                   diseaseUseOnly: Array[Int],
+                                   commercialUseProhibited: Boolean,
+                                   forProfitUseProhibited: Boolean,
+                                   methodsResearchProhibited: Boolean,
+                                   aggregateLevelDataProhibited: Boolean,
+                                   controlsUseProhibited: Boolean,
+                                   genderUseOnly: String,
+                                   pediatricResearchOnly: Boolean,
+                                   IRB: Boolean,
+                                   prefix: String)
+
+  case class StructuredDataResponse(consentCodes: Array[String],
+                                    dulvn: Double, //what the heck is this?
```

Comment:
According to `attribute-definitions.json`:

> Structured Data Use Limitations Version Number

Though that describes it as an integer. Being a version number, we should use an integer (or a string if we're using \<major\>.\<minor\>) rather than a double.